### ISSUE TITLE
Sign with proper agent

### ISSUE THREAD

**Issue by laurentsigal** [waldoapp/waldo-go-cli#4]
Right now, we cannot see anymore what's the version of the agent in https://sentry.io/organizations/waldo/issues/1617599371/events/27f2fc502d464d73a505bb89884c74fa

**Comment by eBardX** [comment]
@laurentsigal 
In the case above, it seems that the failed upload was initiated by a _direct_ Fastlane invocation (outside of any CI). I believe the Sentry invocation is relying _solely_ on the (limited) info provided in the User-Agent string of the error request reporting the the upload failure. My best guess is that this User-Agent string looks like: `Waldo Go CLI v3.0.0` — which is confusing to say the least. The body of the error request _should_ contain the following fields:
```json
"agentName":"Waldo Go Lib",
"agentVersion":"1.0.0",
"wrapperName":"Fastlane",
"wrapperVersion":"3.0.0"
```
This provides more complete info. The only piece missing is the version of Waldo Go CLI that the Fastlane plug-in wraps. Since the Waldo Go CLI binaries are bundled with the Fastlane plug-in it is possible to infer its version if necessary (in this case, 1.0.0).

Long story short, the Sentry-related backend code needs to be updated to use the extended version info available on upload error requests.

**Comment by eBardX** [comment]
Kicking over to @laurentsigal.
